BUILD.debian

```diff
@@ -0,0 +1,17 @@
+# build base docker debian image
+
+load("@bazel_tools//tools/build_defs/docker:docker.bzl", "docker_build")
+
+# Extract .xz files
+genrule(
+    name = "jessie_tar",
+    srcs = ["docker-brew-debian-b39d31635ca26c8b1f3d982090ba8d54167c4d85/jessie/rootfs.tar.xz"],
+    outs = ["jessie_tar.tar"],
+    cmd = "cat $< | xzcat >$@",
+)
+
+docker_build(
+    name = "jessie",
```

Comment:
does envoy require ubuntu (and trusty in particular)? seems like we might want xenial (if we stick with ubuntu).

just looking at the image sizes via pantheon, the envoy and manager images seem to be significantly larger than the mixer image (~60-70 vs 50 MBs). I was already considering moving mixer to jessie-slim to decrease the image size. not sure how important that will become.